client/src/util/dataframe/dataframe.js

```diff
@@ -63,7 +63,13 @@ class Dataframe {
   Constructors & factories
   **/
 
-  constructor(dims, columnarData, rowIndex = null, colIndex = null) {
+  constructor(
+    dims,
+    columnarData,
+    rowIndex = null,
+    colIndex = null,
+    __columnsAccessor = [] // private interface
+  ) {
     /*
     The base constructor is relatively hard to use - as an alternative,
     see factory methods and clone/slice, below.
```

Comment:
add __columnsAccessor to params documentation (line 82, github won't let me add notes to collapsed lines)

Reply:
done